### ISSUE TITLE
Add explicit fedora version to channel chooser widget

### ISSUE THREAD

**Issue by castrojo** [ublue-os/bluefin#1526]
![image](https://github.com/user-attachments/assets/166ba83c-943d-41d6-8b90-bcab928ecd1c)

It'd be nice if we could show the F40, F41, etc versions of Fedora for people who want to lock into a version instead of a rolling tag.

**Comment by dnikoloski** [comment]
Do we want this hard-coded like:
Choose:
```
  date (F40)
  latest (F40)
  gts (F39)
  stable (F40)
```

**Comment by castrojo** [comment]
Yeah but ideally we don't want to have to update each one manually every time if there's a release if possible. 

See also: https://github.com/ublue-os/config/issues/251

**Comment by Malix-off** [comment]
Would be nice to also think about re-ordering the tags like :

> | Name | Fedora Version | Description |
> | - | - | - |
> | _the currently selected one (remove the associated element from the list bellow)_ | _associated version_ | _associated description_ |
> | gts | 40 | Default |
> | stable | 41 | Enthusiasts |
> | Latest | 41 | Testers |
> | Date | Fixed Version | Fixed date release |

**Comment by AtomHare** [comment]
With the new PR there's now fedora versions dynamically fetched shown as `latest (F42)`, `gts (F41)` etc

**Comment by castrojo** [comment]
Thanks for this Atom!
